aspnetcore/fundamentals/index.md

```diff
@@ -71,7 +71,9 @@ The content root is the base path to any content used by the app, such as [Razor
 
 ## Web root
 
-The web root of an app is the directory in the project containing public, static resources, such as CSS, JavaScript, and image files.
+The web root of an app is the directory in the project containing public, static resources, such as CSS, JavaScript, and image files. By default, *wwwroot* is the web root.
+
+For ASP.NET Core code, the tilde character (`~`) represents the web root operator and points to the web root.
```

Comment:
Do not invent the term "web root operator". A path in a .cshtml that starts with `"~/"` (the slash is required) is referred to as a virtual path.

And, what is "ASP.NET Core code"? Do you mean "in a `.cshtml` file"?